templates/CRM/Report/Form/Layout/Table.tpl

```diff
@@ -79,8 +79,8 @@
                     {$l}/if{$r}
                     <tr class="crm-report-sectionHeader crm-report-sectionHeader-{$h}"><th colspan="{$columnCount}">
 
-                        <h{$h}>{$section.title|escape}: {$l}$printValue|default:"<em>none</em>"{$r}
-                            ({$l}sectionTotal key=$row.{$column} depth={$smarty.foreach.sections.index}{$r})
+                        <h{$h}>{$section.title|escape}: {$printValue|default:"<em>none</em>"}
+                            ({sectionTotal key=$row.{$column} depth={$smarty.foreach.sections.index} totals=$sectionTotals})
```

Comment:
This can't possibly be right because it's in a capture section that gets evaluated later. If you remove the $l $r then it gets evaluated during capture. You see code on the screen if you run it.

As to whether it was working before, it does seem a little off. e.g. run a simple contribution detail with section header by country. The first one sort of works, but only knows one currency, but then the second one is blank.

Reply:
`$l` and `$r` variables sounds like someone didn't know about `{ldelim}` and `{rdelim}`